### ISSUE TITLE
Hexchat Gtk3

### ISSUE THREAD

**Issue by misterhsp** [SoniEx2/torxchat#6]
Are there any plans to bring Hexchat to GTK3?

thanks...

**Comment by SoniEx2** [comment]
there are plans to drop gtk2. whether that means going with gtk3 or gtk4 (or something else entirely?) remains to be seen. any reason for gtk3 specifically?

**Comment by misterhsp** [comment]
No, I have no reason for Gtk3. Gtk4 is of course also fine. I only asked
because Gtk2 has been dead for a long time.


...



Am 11.08.24 um 20:09 schrieb Soni L.:

**Comment by misterhsp** [comment]
Are there any plans for when the porting to Gtk3 or 4 will start?

...

**Comment by SoniEx2** [comment]
the plan is to drop GUI toolkits entirely.

HTML is never gonna go deprecated or unmaintained, while GTK or QT would. why play catch up with toolkit versions when you can just drop them altogether?

so: say hello to the `fe-web` branch. it has basically nothing on it so far, but it serves as an official announcement.

**Comment by misterhsp** [comment]
This fork here is just as dead as the original.

...
